eclair-core/src/main/resources/reference.conf

```diff
@@ -78,4 +78,23 @@ eclair {
   max-pending-payment-requests = 10000000
   max-payment-fee = 0.03 // max total fee for outgoing payments, in percentage: sending a payment will not be attempted if the cheapest route found is more expensive than that
   min-funding-satoshis = 100000
+
+  db {
+    driver="org.sqlite.JDBC"
+    mainnet {
+      url = "jdbc:sqlite:/home/chris/.eclair/mainnet"
```

Comment:
This might cause troubles for other users than "chris". You could replace it with "~" but i'm afraid it might break the windows builds. An option is to environment variables as specified [here](https://github.com/lightbend/config#optional-system-or-env-variable-overrides) and use {$HOME} instead of "~" (still windows builds might be affected).

Reply:
Most definitely :+1: I'm going to add the WIP in tag.